templates/about.html

```diff
@@ -3,10 +3,9 @@
 {% block body %}
     <div class="jumbotron text-center">
         <h1>About US</h1>
-        <p class="lead">The Bucket is all about giving you the opportunity to maximise your potential and 
-            get to experience all which you have always dreamt of. It's a proprietary service that elps you keep track of all 
-            the exhilirating activities that not many get to experience. Sign up today and give it a spin.
-        </p>
+            <p class="lead">The Bucket is all about giving you the opportunity to maximise your potential and 
+                get to experience all which you have always dreamt of. Sign up today and give it a spin.
+            </p>
```

Comment:
it looks okay now, preferably use 4 spaces or 2 tabs depending on your settings to indent html code here as an example:
`<p class="lead">
     The Bucket is all about giving you the opportunity to maximise your potential and get 
      to experience all which you have always dreamt of. Sign up today and give it a spin.
   </p>`